src/ssl.c

```diff
@@ -26895,7 +26895,7 @@ int EncryptDerKey(byte *der, int *derSz, const EVP_CIPHER* cipher,
 #endif
         return WOLFSSL_FAILURE;
     }
-    XSTRNCPY((char*)*cipherInfo, info->name, cipherInfoSz);
+    XMEMCPY((char*)*cipherInfo, info->name, cipherInfoSz);
```

Comment:
Whoops, this change causes a buffer overrun reading `info->name`.  The solution is to use regular `XSTRCPY()` so that the terminating null in `info->name` is copied, but `info->name` isn't read beyond its terminating null.